app/js/lib/actions.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+function newAddPageAction( pageName ) {
+	return {
+		type: 'ADD_PAGE',
+		payload: { name: pageName }
```

Comment:
Just a side note: you don't have to group the additional information into `payload`. This would also work perfectly fine:
```.js
function newAddPageAction( pageName ) {
  return {
    type: 'ADD_PAGE',
    name: pageName
  }
}
```
(with the appropriate changes in the reducer).